gflownet.py

```diff
@@ -447,7 +450,7 @@ def __init__(
         self.model = make_mlp(
             [self.env.obs_dim]
             + [args.gflownet.n_hid] * args.gflownet.n_layers
-            + [len(self.env.action_space) + 1]
+            + [(len(self.env.action_space) + 1) * 2]
```

Comment:
Fine for now, but not to self and @nikita-0209: let's aim for having separate models for the forward and backward policies (optionally sharing weights) for more flexibility. The policies should optionally be fixed distributions (e.g. uniform) instead of a neural network, which is practical for the backward policy in some cases.